app/client/screens/Profile/WorkoutHistory.tsx

```
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from "react";
+import { View, Text, ScrollView, Dimensions } from "react-native";
+import { TabView } from "react-native-tab-view";
+import {
+  fetchWorkouts,
+  fetchWorkoutWithDetails,
+} from "@/server/services/workouts";
+import { Workout, Exercise, Set } from "@/types/DatabaseModels";
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)